discovery-provider/src/__init__.py

```diff
@@ -322,6 +322,10 @@ def configure_celery(flask_app, celery, test_config=None):
             "update_user_balances": {
                 "task": "update_user_balances",
                 "schedule": timedelta(minutes=5)
+            },
+            "monitoring_queue": {
+                "task": "monitoring_queue",
+                "schedule": timedelta(seconds=60)
```

Comment:
I think we should add the interval into the config instead of hard coding

Reply:
not sure i agree for this one, but don't feel strongly

Reply:
im not sure if it really makes a difference either. it's still a lot of work to ship a change w/ the config to SPs and i don't think we really want to allow people that much flexibility. do you see this potentially causing some issues happening at a 60s frequency?

Reply:
yeah ok! not a huge deal